packages/ripple-nuxt-tide/modules/event/pages/search.vue

```diff
@@ -100,14 +100,14 @@ export default {
     getComputedFilters () {
       let filterValues = this.tideSearch.getFiltersValues(this.searchForm.filterForm)
       // Test date filter based on start / end fields.
+      if (filterValues.field_event_date_start_value) {
+        const setFilterDate = moment(filterValues.field_event_date_start_value.values)
+        filterValues.field_event_date_start_value.values = setFilterDate.startOf('day').toISOString()
+      }
+
       if (filterValues.field_event_date_end_value) {
         const setFilterDate = moment(filterValues.field_event_date_end_value.values)
         filterValues.field_event_date_end_value.values = setFilterDate.startOf('day').toISOString()
-        filterValues['field_event_date_start_value'] = {
-          operator: 'lte',
-          type: 'date',
-          values: setFilterDate.endOf('day').toISOString()
-        }
```

Comment:
We should not auto-fill start date as it should not matter so long as event end-date happens today or in the future.